ui/txs/noves/useDescribeTxs.tsx

```diff
@@ -0,0 +1,89 @@
+import type { UseQueryResult } from '@tanstack/react-query';
+import _ from 'lodash';
+
+import type { NovesDescribeTxsResponse } from 'types/api/noves';
+import type { Transaction } from 'types/api/transaction';
+
+import type { ResourceError } from 'lib/api/resources';
+
+export interface DescribeTxs {
+  txHash: string;
+  type?: string;
+  description?: string;
+  isLoading: boolean;
+  enabled: boolean;
+}
+
+export interface TransactionWithTranslate extends Transaction {
+  translate?: DescribeTxs;
+}
```

Comment:
Since the field is optional, I think it is safe to add it directely to `Transaction` type. But let's rename it to `transalation` (nouns are better than verbs for property names)